scripts/migration/migrate-linux.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+echo -e "********************************************"
+echo -e "      Ferdium User Data Migration Tool      "
+echo -e "          WARNING: Linux OS Only!           "
+echo -e "********************************************"
+echo -e "\n\nThis tool migrates your user data from Ferdi to Ferdium.\n\n"
+read -p "Do you want to proceed? (y/N) " confirm
+
+case "$confirm" in
+    y|Y|[yY][eE][sS] ) echo -e "Starting...";;
+    n|N|[nN][oO] ) exit 1;;
+    * ) exit 1;;
+esac
+
+FERDI_PATH="$HOME/.config/Ferdi"
```

Comment:
Can you use `uname` to check if the OS is `Darwin`? Then you can also support macos in this same file. 

1. Please rename this file to be `migrate-unix.sh`
2. The profile folder for macos is `$HOME/Library/Application Support/Ferdi` (note the space)
3. Wherever you mention "Linux", this will now change to include macos.

Reply:
I have made changes based on your requests. Please do a test on the MacOS side of the script, as i can't do it myself. Thank you

Reply:
> This is how I would have the suggestion in zsh: `if [ "$(uname)" != "Darwin" ]; then` - in `bash`, the number of square-brackets will change (i think)

since the script itself has to be executed with `sh`, i don't think that there would be any issue regarding syntax. I wrote this script in `sh` syntax and executed said script in `fish` during testing